README.md

```diff
@@ -159,6 +159,20 @@ Set the names of the map themes that should be considered as a list:
 export_settings.mapthemes = ["Robot Theme", "French Theme"]
 ```
 
+#### Custom Project Variables Settings
+
+Set the keys of custom variables that should be considered as a list:
+```py
+export_settings.variables = ["First Variable", "Variable with Structure"]
```

Comment:
```suggestion
export_settings.variables = ["first_variable", "other_variable"]
```

Variables normally do not have spaces (so they can be used with `@first_variable`) if I'm not wrong?

Reply:
Yeah. You are completely right. I think the toppingmaker should not slugify anything here, but the README example should show the correct behavior. I don't commit here, but change it everywhere instead (like in tests etc.).

**Played around and for the record:**
I the QGIS Project Property when I type "First Variable" it's stored as "First_Variable", so one cannot enter spaces in the keys. 

With PyQGIS this works:
```py
QgsExpressionContextUtils.setProjectVariable(QgsProject.instance(),'Space Variable',"spaces in the space")
QgsExpressionContextUtils.projectScope(QgsProject.instance()).variable('Space Variable')
'spaces in the space'
```
...but the variables are not usable in the expressions:
![grafik](https://user-images.githubusercontent.com/28384354/206127746-018c53c9-7ff0-4def-bd96-4007e14181d4.png)

Reply:
FTR: Pretty sure `var('Space Variable')` would work, but I don't assume anyone uses it.
Fine for me to adjust it just for the public doc / readme

Reply:
![grafik](https://user-images.githubusercontent.com/28384354/206130119-7d74afb7-596d-42b1-a030-4487257b0d88.png)

Reply:
Another note. In the Project Properties you only can add strings but we do support structures like e.g. lists. Since - compared to the spaces - this provides more possibilities we support it and have it here in the example.

Reply:
![image](https://user-images.githubusercontent.com/588407/206130502-69941ea8-5f15-4ca8-bed5-68a041cf247d.png)

Reply:
Voll space-ig :-)

Reply:
> If you do that and then go to project properties and accept the dialog, are those preserved as structures?

Yes.

![grafik](https://user-images.githubusercontent.com/28384354/206131925-c82a02ea-2a86-4286-b676-4bf6e193df4d.png)

But: After storing the project to the .qgs and open it again, it's gone. I guess `writeXML` (or `readXML`) cannot handle it :disappointed:

Reply:
Dommage - I already dreamed about awesome possibilities in expressions.
So I guess I remove it from the docs as well.

Reply:
I'd rather remove it from the doc in this case (and keep it in mind as potential for a QGIS improvement)